src/scss/SeriesSearch.scss

```diff
@@ -1,7 +1,46 @@
 .search-form {
   text-align: center;
+  & input {
+    box-sizing: border-box;
+    font-weight: bold;
+    border-color: black;
+    border: 1px solid;
+    width: 50vw;
+    padding: 10px;
+    height: 2rem;
+  }
 }
 
-.series-search h1 {
-  font-size: 1rem;
+.search-results-overlay {
+  position: absolute;
+  left: 25vw;
+  background: tomato;
+  border-radius: 3px;
+  border: none;
+  width: 50vw;
+}
+
+.search-series-suggestion {
+  display: flex;
```

Comment:
i expected to have an alternating background-color for these suggestions. maybe even ones are orang and odd ones are dark orange